### DISCUSSION TITLE
Color theme in code blocks

### DISCUSSION THREAD

**Discussion by devrnt** [community/community#22191] [Accessibility]
<p>Github dark mode looks so great, especially the color theme for the code blocks.</p>
<p>Which code theme is used in the Github dark mode for the <strong>code blocks</strong> or is this custom made for the dark mode?</p>
<p>I can’t seem to find any information or already existing themes.</p>
<p>Thanks in advance!</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Hi <a class="mention" href="/devrnt">@devrnt</a> - thanks for reaching out! Dark mode for GitHub is a completely in-house custom design and implementation. Let us know if you have any feedback on it!</p>

**Comment by devrnt** [comment]
<p>Hi <a class="mention" href="/gaknoll">@gaknoll</a> thanks for the response.</p>
<p>It’s looking great! I hoped this color theme would be available in VS Code as a color theme. Maybe one day in the future :slight_smile:</p>

**Comment by cehfisher** [comment]
Hi @devrnt - this issue seems to have been resolved [per your last response](https://github.com/orgs/community/discussions/22191#discussioncomment-3235819). Could you please confirm and close this issue? Thanks! (cc @queenofcorgis)
